### ISSUE TITLE
Weapon dupe

### ISSUE THREAD

**Issue by Callum-Doggo** [rebel1324/BlackTea-Nutscript-Plugins#3]
When I have a CW: 2.0 weapon equipped and I die, I drop the weapon after death but when I get revived (advanced medic mod is added), the weapon is on the floor but still on the scroll wheel. I can load and fire the weapon like normal, so there's 2 guns, one on the floor which I can pick up and one on my person which I can't get rid of.

**Comment by rebel1324** [comment]
Are you using old version of CityRP

**Comment by Callum-Doggo** [comment]
No, I used HL2RP but I have the CWsupport plugin from CityRP. Should I switch Schema?
